### ISSUE TITLE
Way to feed global mouse position into HTML based wallpapers?

### ISSUE THREAD

**Issue by GanerCodes** [rocksdanister/lively#813]
Not sure how to go about it, but some way to give your mouse position to your wallpaper [even if it's in another display] would be nice

**Comment by rocksdanister** [comment]
Can you list a use case for this?

**Comment by GanerCodes** [comment]
https://user-images.githubusercontent.com/34655998/137627566-536db5c3-154d-4f82-ae52-d3f8c9b77b35.mp4

**Comment by rocksdanister** [comment]
I know this is not a direct solution but if you span the wallpaper across both displays and position the ghost accordingly, you will get global coordinates.

Global coordinates are converted to per-display before passing unless its span mode:
https://github.com/rocksdanister/lively/blob/2ea813a7e3e9a2f1298167c139464897bba3d0e5/src/livelywpf/livelywpf/Core/InputForwarding/RawInputDX.xaml.cs#L300-L324

There is no other way to do this unless a custom support is added:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data

**Comment by GanerCodes** [comment]
Yes however I have 6 monitors and I don't want the browser environment to span across all of them for performance/practicality reasons. Is there some way to apply directly communication? In wallpaper engine you can activate the executable with certain args that get passed into the wallpaper

edit: it seems you already do custom stuff for system properties, could you not just read the mouse info and pass that along in a similar way as system specs? or would that cause lag or something

**Comment by rocksdanister** [comment]
>Is there some way to apply directly communication?

https://github.com/rocksdanister/lively/wiki/Command-Line-Controls
https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#lively-properties

>it seems you already do custom stuff for system properties, could you not just read the mouse info and pass that along in a similar way as system specs? or would that cause lag or something

Not really, its just work..

**Comment by GanerCodes** [comment]
oh, do you plan to do it sometime soon?

**Comment by rocksdanister** [comment]
I am in the middle of a big rewrite: 
https://github.com/rocksdanister/lively/tree/Refactoring-2.0

So not anytime soon.

**Comment by GanerCodes** [comment]
understandable, will you throw it into there when u add the pc spec integration or nah?

**Comment by rocksdanister** [comment]
PC spec is already ready 😉:
https://github.com/rocksdanister/system-stats-wallpaper
https://github.com/elias123tre/lively-audio-visualizer

> will you throw it into there

Sure!

**Comment by GanerCodes** [comment]
Thanks man, awesome!

**Comment by rocksdanister** [comment]
Keep it open.
